Makefile

```diff
@@ -1,38 +1,25 @@
-CC=clang
-CXX=clang
-LD=ld.lld
-RUST=rustc
-
-CFLAGS=$(shell llvm-config --cflags) -emit-llvm -Iinclude
-CXXFLAGS=$(shell llvm-config --cxxflags) -emit-llvm
-LDFLAGS=$(shell llvm-config --ldflags)
-RUSTFLAGS=--crate-type=lib --emit=llvm-bc
-
-LDLIBS=stdc++
-OBJECTS=$(addprefix build/, $(addsuffix .ll, hello.c hello.cpp hello.rs main.c))
-
-.PHONY: all
-all: link
-
-.PHONY: link
-link: build
-	$(LD) $(LDFLAGS) $(addprefix -l, $(LDLIBS)) -o hello $(OBJECTS)
-
-.PHONY: build
-build: build/ $(OBJECTS)
-
-.PHONY: clean
-clean:
-	$(RM) $(OBJECTS) hello
-
-build/:
+.PHONY:
```

Comment:
I didn't know you could use .PHONY like that

Reply:
It might be specific to GNU Make, I'm not sure. Let me know if there's another Make you want to support.